tests/common.py

```diff
@@ -131,17 +131,7 @@ def prepare_lj_system(
 def prepare_reference_nonbonded(params, exclusion_idxs, scales, lambda_plane_idxs, lambda_offset_idxs, beta, cutoff):
 
     N = params.shape[0]
-
-    # process masks for exclusions properly
-    charge_rescale_mask = np.ones((N, N))
-    for (i, j), exc in zip(exclusion_idxs, scales[:, 0]):
-        charge_rescale_mask[i][j] = 1 - exc
-        charge_rescale_mask[j][i] = 1 - exc
-
-    lj_rescale_mask = np.ones((N, N))
-    for (i, j), exc in zip(exclusion_idxs, scales[:, 1]):
-        lj_rescale_mask[i][j] = 1 - exc
-        lj_rescale_mask[j][i] = 1 - exc
+    charge_rescale_mask, lj_rescale_mask = nonbonded.convert_exceptions_to_rescale_masks(exclusion_idxs, scales, N)
```

Comment:
Point of clarification, is an exception an exclusion?

Reply:
Renamed in https://github.com/proteneer/timemachine/pull/685/commits/c4077aeb359ff1179e6f44d837e610a61ccdfc81